### PR TITLE
Improve audioplayer cleaned#140

### PR DESCRIPTION
**Pull request by vandrongelen**

### CODE REVIEW

File: src/components/AudioPlayer.vue [hunk 1]
```diff
@@ -1,87 +1,122 @@
...
-                icon
-                @click="closePlayer"
-              >
-                <v-icon>mdi-close</v-icon>
-              </v-btn>
-            </v-list-item-icon>
-          </v-list-item>
-        </v-list>
+                <div
+                  class=""
```
Comment (dokterbob) [inline src/components/AudioPlayer.vue:65]: Perhaps best to remove empty class, as it doesn't do anything.

File: src/components/AudioPlayer.vue [hunk 2]
```diff
@@ -95,13 +130,22 @@ export default {
   beforeDestroy() {
     this.audioPlayer.close();
   },
+  data() {
+    return {
+      // isSmallPlayer: true,
```
Comment (dokterbob) [inline src/components/AudioPlayer.vue:135]: It seems that this is not used anywhere, perhaps best to remove it?

File: src/components/AudioPlayer.vue [hunk 3]
```diff
@@ -95,13 +130,22 @@ export default {
...
+      playerHeight: 100,
```
Comment (dokterbob) [inline src/components/AudioPlayer.vue:136]: The value of this is only ever `100`. Is this intended behaviour?

File: src/components/AudioPlayer.vue [hunk 4]
```diff
@@ -1,87 +1,122 @@
...
-      transition="slide-y-transition"
+    <v-footer
       v-if="playerActive"
       v-model="playerActive"
-      class="audio-player"
       dark
-      height="100"
-      hide-overlay
-      persistent
+      height="playerHeight"
```
Comment (dokterbob) [inline src/components/AudioPlayer.vue:7]: Shouldn't this be `:height="playerHeight"`, as it is in `<v-card>` below?

File: src/components/AudioPlayer.vue [hunk 5]
```diff
@@ -1,87 +1,122 @@
...
+      fixed
+      padless
+      style="z-index: 10000;"
     >
       <v-card
+        class="audio-player-card"
         tile
         width="100%"
         :height="playerHeight"
```
Comment (dokterbob) [inline src/components/AudioPlayer.vue:16]: Here it's `:height="playerHeight"` while in `<v-footer>` above it's `height="playerHeight"`.

Perhaps we want to be consistent here?

File: src/components/AudioPlayer.vue [hunk 6]
```diff
@@ -1,87 +1,122 @@
...
+                    :src="sourceFile.src"
+                  >
+                    <v-icon
+                      size="32"
+                      color="white"
+                      style="opacity: 0.3;
+                        position: absolute;
+                        top: 50%;
+                        left: 50%;
+                        transform: translate(-50%, -50%);"
```
Comment (dokterbob) [inline src/components/AudioPlayer.vue:50]: Just a suggestion, but perhaps the code will look cleaner and will be easier to maintain by moving more static css to the `<style>` section?

**Review by dokterbob — CHANGES_REQUESTED** [review]
Works like a charm!

Some small cleanup would be nice though. :)

I solved a merge conflict during testing and pushed it back. Be sure to pull before cleaning up.

Could I please rebase on merging?

File: src/components/AudioPlayer.vue (hunk 2 above)
Comment (vandrongelen) [reply src/components/AudioPlayer.vue:135]: Done

File: src/components/AudioPlayer.vue (hunk 1 above)
Comment (vandrongelen) [reply src/components/AudioPlayer.vue:65]: Done

File: src/components/AudioPlayer.vue (hunk 3 above)
Comment (vandrongelen) [reply src/components/AudioPlayer.vue:136]: This is for later usage.

**Review by dokterbob — APPROVED** [review]

**Pull request merged**